### ISSUE TITLE
[Incompatibility] Corporate Restructure

### ISSUE THREAD

**Issue by NilaierMusic** [Shaosil/LethalCompanyMods-GeneralImprovements#44]
New config is great for customization, but there's a bit of an issue when you try to remove blue background from extra monitors while having [Corporate Restructure](https://thunderstore.io/c/lethal-company/p/Jamil/Corporate_Restructure/) mod installed as well. Corporate Restructure breaks completely while extra monitor from GeneralImprovements defaults to a quota monitor still having a blue background.
![IMG_20240119_043632](https://github.com/Shaosil/LethalCompanyMods-GeneralImprovements/assets/86863646/34bbbf57-3ff1-4343-81c2-d313c6716557)
There's a couple of errors that pop up along with it as well:
```cs
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
GeneralImprovements.Utilities.SceneHelper.CreateExtraMonitors () (at <3f608587e5564bd68f727547011be618>:0)
GeneralImprovements.Patches.StartOfRoundPatch.Start (StartOfRound __instance) (at <3f608587e5564bd68f727547011be618>:0)
(wrapper dynamic-method) StartOfRound.DMD<StartOfRound::Start>(StartOfRound)
```
```cs
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
CorporateRestructure.Component.CreditMonitor.UpdateMonitor () (at <145f04274b9a4d6b9e59d1553ca40844>:0)
CorporateRestructure.Patch.MonitorPatch.OnPlayerConenct () (at <145f04274b9a4d6b9e59d1553ca40844>:0)
(wrapper dynamic-method) GameNetcodeStuff.PlayerControllerB.DMD<GameNetcodeStuff.PlayerControllerB::ConnectClientToPlayerObject>(GameNetcodeStuff.PlayerControllerB)
GameNetcodeStuff.PlayerControllerB.Update () (at <af9b1eec498a45aebd42601d6ab85015>:0)
```

**Comment by Shaosil** [comment]
Oof I really wanted to test it with that mod but I was rushing to get it out before this evening lol. I'll fix that as soon as I can.

**Comment by Shaosil** [comment]
Fixed in v1.0.24. Although I should note that CorporateRestructure still displays that bug even without my mod loaded, which was actually preventing some of my code from running. Might be worth bringing it up to them.
